dockstore/dockstore-ui2
# Remove metrics feature flag

**Description**
Removes the metrics feature flag from the UI.

The versions tab is crowded if both the Verified Platforms and Metrics columns are present, which creates a scroll bar to access the Actions column. Previously, the Verified Platforms column was hidden if the metrics feature flag was on.

This PR turns the Verified Platforms column into a Verified column where the value is a check mark if the version is verified and moves the Verified Platforms information into the Info dialog.

**Before:**
![image](https://github.com/dockstore/dockstore-ui2/assets/25287123/b2a2a62b-004a-472d-a6e5-e63b07e377d8)

**After:**
![Screenshot 2024-01-16 at 12-07-33 Dockstore Workflow github com_DataBiosphere_topmed-workflows_UM_variant_caller_wdl](https://github.com/dockstore/dockstore-ui2/assets/25287123/24631752-9920-4868-a0e3-6fa13c9530c0)

In the Info dialog, if the version is Verified, then Verified Platforms are displayed.
![Screenshot 2024-01-16 at 12-11-03 Dockstore Workflow github com_alexanderhsieh_test_read_tsv_test_read_tsv github com_DataBiosphere_topmed-workflows_UM_variant_caller_wdl](https://github.com/dockstore/dockstore-ui2/assets/25287123/42fa3865-7685-4235-8f3a-8e58744eeaa2)

If the version is not Verified, then Verified Platforms are not shown.
![Screenshot 2024-01-16 at 12-11-16 Dockstore Workflow github com_alexanderhsieh_test_read_tsv_test_read_tsv github com_DataBiosphere_topmed-workflows_UM_variant_caller_wdl](https://github.com/dockstore/dockstore-ui2/assets/25287123/c87f6733-8118-4dfe-85fc-4fe976f1dad7)


<details>
<summary>Old PR description</summary>
The versions tab is pretty crowded now and the Actions button is hidden unless scrolled to:
![Screenshot 2024-01-12 at 10-22-52 Dockstore Workflow github com_theiagen_terra_utilities_Concatenate_Column_Content](https://github.com/dockstore/dockstore-ui2/assets/25287123/a4a99073-d914-4f77-9b09-dffac0ef18d7)

![Screenshot 2024-01-12 at 10-23-08 Dockstore Workflow github com_theiagen_terra_utilities_Concatenate_Column_Content](https://github.com/dockstore/dockstore-ui2/assets/25287123/a6bfc3f6-629d-4545-add7-848527e9b619)

Previously we hid the Verified Platforms column if the metrics flag was enabled. Should we permanently hid the Verified Platforms column in favour of the metrics column?

</details>

**Review Instructions**
During these steps, ensure that there is no "preview" banner.

Verify that you can:
- View the metrics facet on the search page
- select a workflow with metrics from the search page
- Click on the versions tab and verify that the metrics column is displayed
- Select a version with metrics and go to the Metrics tab
- Verify that the metrics are displayed

**Issue**
[SEAB-6090](https://ucsc-cgl.atlassian.net/browse/SEAB-6090)

**Security**
If there are any concerns that require extra attention from the security team, highlight them here.

Please make sure that you've checked the following before submitting your pull request. Thanks!



[SEAB-6090]: https://ucsc-cgl.atlassian.net/browse/SEAB-6090?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
